src/appleseed.cli/stdouttilecallback.cpp

```diff
@@ -67,6 +68,12 @@ namespace
       : public TileCallbackBase
     {
       public:
+        StdOutTileCallback(const bool single_plane = false)
```

Comment:
Mark the constructor as `explicit` to avoid implicit conversions from `bool`.

The code would be even clearer with two changes:
- Use an enum rather than a boolean parameter
- Remove the default value

Reply:
I'm confused about another thing: are there cases where `single_plane` is set to false? I can only see a single use of this constructor and `single_plane` is set to true.

Reply:
No there isn't any such case for the moment. I have to modify the factory. And also, find a way to make this optional from the cli.